tests/test_flow/test_persistence_random.py

```diff
@@ -37,24 +38,32 @@ class ModelFlowHarness:
     - this class does not support the following:
       - non-deterministic (``changes_per_run``) entities
       - non-functional updates
-      - alternative versioning modes
       - cloud caching
 
     This class has a more sophisticated implementation than SFM, so it should be
     easier to port SFM's features to this class than vice versa. If we can port all of
     the above features, we can remove SFM altogether.
     """
 
-    def __init__(self, builder, make_list, expect_exact_call_count_matches=False):
+    def __init__(
+        self,
+        builder,
+        make_list,
+        parallel_execution_enabled=False,
+        expect_exact_call_count_matches=False,
+    ):
         self._builder = builder
         self._versioning_mode = "manual"
+        self._parallel_execution_enabled = parallel_execution_enabled
         self._expect_exact_call_count_matches = expect_exact_call_count_matches
 
         self._entities_by_name = {}
 
         self._descriptors_computed_by_flow = make_list()
         self._descriptors_computed_by_model = make_list()
 
+        self._clear_cur_flow()
```

Comment:
I like "latest" less than "current"; I think the latter is more precise. ( I feel like something could be "latest" but not "current", but not vice versa.) Do you think "current" would be better than "cur"?

Reply:
I think my issue is I don't really understand what a "current flow" is (I understand now, but I'm sure I'll have to read the code again later to figure out what it is.) Maybe adding a comment about what it is would help.

My understanding is that `_cur_flow` is the "up-to-date flow with latest bindings". You probably shouldn't use that as a name lol but maybe you can think of a better name from the description. 🤷🏽‍♂️

Reply:
Hmm! I just think of it as the flow that corresponds to the model. Maybe "current" is redundant here: do you think it would be clearer if we just called it `_flow`?

Reply:
Yeah, I think that is easier to understand.

Reply:
After some off-PR discussion, we settled on `active_flow`, in order to preserve the correspondence between `_active_flow` and `__active_model_memoized_values_by_entity_name`.